pyclam/manifold.py

```diff
@@ -163,7 +167,7 @@ def argmedoid(self) -> int:
         """ The index used to retrieve the medoid. """
         if '_argmedoid' not in self.__dict__:
             logging.debug(f"building cache for {self}")
-            _argmedoid = np.argmin(cdist(self.samples, self.samples, self.metric).sum(axis=1))
+            _argmedoid = np.argmin(self.distance(x1=self.argsamples, x2=self.argsamples).sum(axis=1))
```

Comment:
same here, would recommend against using keyword arguments for functions where the order doesn't matter

Reply:
https://www.python.org/dev/peps/pep-0570/